design/general_refactorings.md

```diff
@@ -0,0 +1,49 @@
+Before refactor concrete thing here is a list of things that need to be done all over the code:
+
+1. Go Over All Namings And Improve them
+
+2. Directory Organize:
+
+   Current Directories
+
+   | Directory Name | Porpuse                           |
+   | :------------- | :-------------------------------- |
+   | host           | files related to host command     |
+   | init           | files related to init command     |
+   | list           | files related to list command     |
+   | login          | files related to login command    |
+   | logout         | files related to logout command   |
+   | policy         | files related to policy command   |
+   | user           | files related to user command     |
+   | variable       | files related to variable command |
+
+   
```

Comment:
The following files are without parent directories and sit in the main directory...

Reply:
Done